rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/service/DataSourceService.java

```diff
@@ -27,17 +27,16 @@ public DataSourceService(GraphContext context)
     public synchronized DataSourceModel createUnique(Set<ProjectModel> applications, String dataSourceName, String jndiName)
     {
         JNDIResourceModel jndiResourceModel = new JNDIResourceService(getGraphContext()).createUnique(applications, jndiName);
-        final DataSourceModel dataSource;
+        final DataSourceModel dataSourceModel;
         if (jndiResourceModel instanceof DataSourceModel)
```

Comment:
This old code is a little unclear to me, as DataSourceModel is not here as it is always created as JNDIResourceModel

Reply:
I think it is possible for this method to be called more than once. It may be that the DataSourceModel type has already been added on later calls.

Reply:
ok, I still can't get it to work with https://github.com/windup/windup-rulesets/pull/270, any idea? Am I overlooking anything obvious?